config/sync/core.entity_view_display.node.contact.full.yml

```diff
@@ -160,6 +159,11 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  toc_display:
```

Comment:
Same here, I don't think we need the toc_display , they should be in the hidden config.